src/v2/services/IXcmService.ts

```diff
@@ -1,5 +1,5 @@
-import { Asset } from 'src/v2/models';
+import { XcmAssets } from './../../store/assets/state';
```

Comment:
just can go through `src/store/assets/state` ?

Reply:
I have fixed in this commit https://github.com/AstarNetwork/astar-apps/pull/469/commits/ed79f2ae9bf5c316d52f3683f83024e8673eb5ca